### DISCUSSION TITLE
What is the difference

### DISCUSSION THREAD

**Discussion by Cansouh** [staxrip/staxrip#873] [Q&A]
What is the difference  x264 Patman Mod and x264 Original

**Comment by jessielw** [comment]
It is specifically compiled for staxrips needs

**Reply by Cansouh** [reply]
Please ,Can you give me staxrip that encode with x264 orginal and x265 orginal
I

**Reply by jessielw** [reply]
I'm not developer, but you can just put the originals in the apps folder of the program

**Reply by Cansouh** [reply]
> I'm not developer, but you can just put the originals in the apps folder of the program

When i do this avs dosen't work 100% 
 And this is a broblem 
https://github.com/staxrip/staxrip/issues/871#issue-1224749831

**Reply by jessielw** [reply]
Pretty sure you need to change the pipe method in the x264 settings

**Reply by mr-qansuh** [reply]
> Pretty sure you need to change the pipe method in the x264 settings

how can i change it

**Reply by jessielw** [reply]
The error here 
https://k.top4top.io/p_2314tbt1m1.jpg

Shows you where and how to remedy this in the screenshot. That's the same place you can change the pipe method.

**Comment by Dendraspis** [comment]
This is a modded x264 build with some additional features like the direct AVS and VS reading ability and some small fixes.

I don't know why anybody should want to use another build, but you can simply change the path of the x264 app, so you can keep the original structure of StaxRip while using a different build.

**Reply by Cansouh** [reply]
x264 Patman Mod size is 6 M
x264 Original size is 17 M

Is there a difference in quality of video after encoding

**Reply by Dendraspis** [reply]
Nope
